test/api_test.js

```diff
@@ -130,4 +143,25 @@ suite('api', () => {
     result = await helper.github.isInstalledFor('unknownOwner', 'unknownRepo');
     assert.deepEqual(result, {installed: false});
   });
+
+  test('build badges', async function() {
+    await request.get('http://localhost:60415/v1/badge/abc123/coolRepo/master').end((err, res) => {
+      err ? console.log(err) : assert.equal(res.headers['content-length'], 928);
```

Comment:
Yeah, it's a little bit weird. We don't want to `console.log`. I'll comment here again soon when I figure out how to do this.